toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -255,9 +259,13 @@ func (r *RpmRepoCloner) initializeRepoDefinitions(repoDefinitions []string) (err
 	// Add each previously existing repofile to the end of the new file, then delete the original.
 	// We want to try our custom mounted repos before reaching out to the upstream servers.
 	for _, originalRepoFilePath := range existingRepoFiles {
-		err = appendRepoFile(originalRepoFilePath, dstFile)
-		if err != nil {
-			return
+		if !disableMarinerRepos {
```

Comment:
Could we add a bit more to the comment here explaining how the repos will get ordered and added/removed with this flag? Its a bit magic right now. (At least mention that the chroot ships with the mariner repos and we want to remove them).